cmd/olympus/actions/merge_db.go

```diff
@@ -1,14 +1,16 @@
 package actions
 
 import (
+	"context"
 	"io/ioutil"
 	"log"
 	"time"
 
-	"github.com/gomods/athens/pkg/storage"
-
 	"github.com/gomods/athens/pkg/cdn"
 	"github.com/gomods/athens/pkg/eventlog"
+	"github.com/gomods/athens/pkg/storage"
+
```

Comment:
newline please

Reply:
Sorry, I'm not sure what you're speaking to here. I have it organized into three clauses separated by newlines:
1. stdlib
1. project internal packages
1. external packages

Would you like me to consolidate some of those clauses? Break them up differently? Maybe add a newline after the last clause?

Reply:
oh i see. i used stdlib / anything else approach so far

Reply:
as internal external it's still github path

Reply:
Makes sense :) I'll tweak it to stay consistent unless you say otherwise.

Reply:
Fixed locally